python/py-serial/Portfile

```diff
@@ -25,7 +25,7 @@ checksums           rmd160  ee56e22c15af484b72b65d445a4fe8a2aeee0cd0 \
                     sha256  80125f950a620bd0a31cfa18f5eda0df77e0bed2accc2bb7a76ab650ff63afa9 \
                     size    155864
 
-python.versions     27 34 35 36 37 38 39 310 311 312
+python.versions     27 34 35 36 37 38 39 310 311 312 313
```

Comment:
please remove the subports for EOL Python versions that have no dependents in MacPorts

Reply:
I really wish people would stop obsessing over "EOL".  Older Python versions shouldn't be subject to aggressive elimination any more than older OS versions (most of which are definitely "EOL"), and supporting older OS versions is one of the things that sets MacPorts apart from Homebrew.  MacPorts is, in principle, an excellent platform for testing code against a wide range of Python versions, as long as that capability isn't destroyed by EOLism.

Most Python versions are currently available for most, if not all, OS/CPU combinations, with `python313` being the worst offender (presumably due to newness).  Specifically:

`python36` doesn't build on 10.4 ppc
`python37` doesn't build on 10.4 ppc, 10.5 ppc, or 10.5 x86_64
`python312` _now_ doesn't build on 10.7 x86_64, but former versions did
`python313` doesn't build on 10.4 *, 10.5 *, or 10.7 x86_64

Of the 253 relevant Python/OS/CPU combinations that I'm able to test, 10 are currently unavailable due to broken Python builds (more than half of which are `python313`), and the remaining 243 were successfully tested with `py-serial` yesterday.

BTW, Python3.4 was the last version with a sane interaction between signals and select()/poll().  This is just one example of how newer isn't always better.

Reply:
well.... we certainly are not going to agree on this but it's not worth my time arguing about it